src/components/barchart/BarChart.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react'
+import ReactDOMServer from 'react-dom/server'
+import { ResponsiveBar } from '@nivo/bar'
+import { Card } from '../card'
+import { Table } from 'antd'
+
+import Highcharts from 'highcharts'
+import HighchartsReact from 'highcharts-react-official'
+
+type Props = {
+  library: 'nivo' | 'highcharts'
+}
+
+const BarChart = ({ library }: Props) => {
+  switch (library) {
+    case 'nivo':
+      return <NivoBarChart />
+    case 'highcharts':
+      return <HighchartsBarChart />
+    default:
+      return null
+  }
+}
+
+const HighchartsBarChart = () => {
+  const data = [
+    {
+      plan: 'SCAN-HMO-01',
+      providerShare: 0.0789 * 100,
+      totalProviders: 10000,
+    },
+    {
+      plan: 'SCAN-HMO-02',
+      providerShare: 0.1234 * 100,
+      totalProviders: 10000,
+    },
+    {
+      plan: 'UHC-HMO-05',
+      providerShare: 0.1588 * 100,
+      totalProviders: 10000,
+    },
+  ]
+
+  console.log(data.map((item) => item.plan))
+
+  const options: Highcharts.Options = {
+    chart: {
+      type: 'column',
+    },
+    xAxis: {
+      categories: data.map((item) => item.plan),
+      title: {
+        text: 'Network Plan',
+        style: {
+          fontSize: '14px',
+          fontFamily: 'Rubik',
+          color: '#20201B',
+        },
+      },
+      labels: {
+        y: 30,
+        style: {
+          fontSize: '14px',
+          fontFamily: 'Rubik',
+          color: '#20201B',
+        },
+      },
+    },
+    yAxis: {
+      min: 0,
+      max: 100,
+      title: {
+        text: 'Total Providers',
+        style: {
+          fontSize: '14px',
+          fontFamily: 'Rubik',
+          color: '#20201B',
+        },
+      },
+      tickInterval: 10,
+      labels: {
+        format: '{value}%',
+        style: {
+          fontSize: '14px',
+          fontFamily: 'Rubik',
+          color: '#20201B',
+        },
+      },
+    },
+    tooltip: {
+      useHTML: true, // Enable HTML rendering for the tooltip
+      formatter: function () {
+        const plan = this.x
+        const providerShare = this.y as number
+        const columns = [
+          {
+            title: 'Plan',
+            dataIndex: 'plan',
+            key: 'plan',
+          },
+          {
+            title: 'Provider Share',
+            dataIndex: 'providerShare',
+            key: 'providerShare',
+          },
+        ]
+
+        const data = [
+          {
+            key: '1',
+            plan: plan,
+            providerShare: `${providerShare.toFixed(2)}%`,
+          },
+        ]
+
+        return ReactDOMServer.renderToString(
+          <Card>
+            <Table columns={columns} dataSource={data} pagination={false} />
+          </Card>
+        )
```

Comment:
Damn that's a bummer you have to do this. This can kinda limit our ability to make interactive tooltips, hmmm